R/pkg/R/DataFrame.R

```diff
@@ -2714,6 +2714,40 @@ setMethod("unionAll",
             union(x, y)
           })
 
+#' Return a new SparkDataFrame containing the union of rows
```

Comment:
I think I'd suggest a slight different in the title - this one is the same words as `union`

Reply:
Sure, I just addressed both comments.